mlflow/tracking/client.py

```diff
@@ -3386,6 +3386,9 @@ def set_registered_model_alias(self, name: str, alias: str, version: str) -> Non
         """
         Set a registered model alias pointing to a model version.
 
+        .. Note:: Aliases of the format ``v<number>``, such as ``v9`` and ``v42``, are reserved
+                  and cannot be set.
```

Comment:
Should this be included in the `alias` param description?

Reply:
Good idea. Let me move it there.